traffic_ops/install/bin/install_goose.sh

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 #
 
-trap "echo Error on line ${LINENO} of '$0'; exit 1" ERR
+trap 'echo "Error on line ${LINENO} of ${0}"; exit 1' ERR
```

Comment:
Why?

Reply:
`$LINENO` and `$0` are supposed to be expanded when a non-conditional line returns a nonzero exit code, which evaluates the ERR signal trap. Before, they were being set when the script started.

Reply:
Was `LINENO`? because `$0` shouldn't change during script execution

Reply:
Yep $LINENO is the current line that is executing, and to get an accurate value in the trap, it needs to expand no sooner than a line returns an error.

Reply:
Alright, I'll take your word for it